diff --git a/examples/makefile_Unix b/examples/makefile_Unix
index 01ad99c..745beab 100644
--- a/examples/makefile_Unix
+++ b/examples/makefile_Unix
@@ -40,44 +40,44 @@
 
 include ../make_Unix.inc
 
-all: AB01MD.exa AB01ND.exa AB01OD.exa AB04MD.exa AB05MD.exa AB05ND.exa \
+all: AB05MD.exa AB05ND.exa \
      AB05OD.exa AB05PD.exa AB05QD.exa AB05RD.exa AB07MD.exa AB07ND.exa \
-     AB08ND.exa AB08NW.exa AB08NZ.exa AB09AD.exa AB09BD.exa AB09CD.exa \
+     AB09AD.exa AB09BD.exa AB09CD.exa \
      AB09DD.exa AB09ED.exa AB09FD.exa AB09GD.exa AB09HD.exa AB09ID.exa \
      AB09JD.exa AB09KD.exa AB09MD.exa AB09ND.exa AB13AD.exa AB13BD.exa \
-     AB13CD.exa AB13DD.exa AB13ED.exa AB13FD.exa AB13ID.exa AB13MD.exa \
-     AG08BD.exa AG08BZ.exa \
+     AB13DD.exa AB13ED.exa AB13FD.exa AB13MD.exa \
+     \
      BB01AD.exa BB02AD.exa BB03AD.exa BB04AD.exa BD01AD.exa BD02AD.exa \
      DE01OD.exa DE01PD.exa DF01MD.exa DG01MD.exa DG01ND.exa DG01OD.exa \
      DK01MD.exa \
      FB01QD.exa FB01RD.exa FB01SD.exa FB01TD.exa FB01VD.exa FD01AD.exa \
-     IB01AD.exa IB01BD.exa IB01CD.exa IB03AD.exa IB03BD.exa \
+     IB01AD.exa IB01BD.exa IB01CD.exa IB03AD.exa \
      MB01TD.exa MB02CD.exa MB02DD.exa MB02ED.exa MB02FD.exa MB02GD.exa \
-     MB02HD.exa MB02ID.exa MB02JD.exa MB02JX.exa MB02KD.exa MB02MD.exa \
-     MB02ND.exa MB02QD.exa MB02SD.exa MB02VD.exa MB03BD.exa MB03BZ.exa \
-     MB03FZ.exa MB03KD.exa MB03LD.exa MB03LF.exa MB03LZ.exa MB03MD.exa \
+     MB02HD.exa MB02ID.exa MB02JD.exa MB02KD.exa MB02MD.exa \
+     MB02ND.exa MB02QD.exa MB02SD.exa MB03BD.exa \
+     MB03KD.exa MB03LD.exa MB03MD.exa \
      MB03ND.exa MB03OD.exa MB03PD.exa MB03QD.exa MB03QG.exa MB03RD.exa \
      MB03SD.exa MB03TD.exa MB03UD.exa MB03VD.exa MB03WD.exa MB03XD.exa \
-     MB03XP.exa MB03XZ.exa MB03ZD.exa MB04AD.exa MB04AZ.exa MB04BD.exa \
-     MB04BZ.exa MB04DD.exa MB04DL.exa MB4DLZ.exa MB04DP.exa MB4DPZ.exa \
-     MB04DS.exa MB04DY.exa MB04DZ.exa MB04ED.exa MB04FD.exa MB04GD.exa \
+     MB03XP.exa MB03ZD.exa MB04AD.exa MB04BD.exa \
+     MB04DD.exa MB04DL.exa MB04DP.exa \
+     MB04DS.exa MB04DY.exa MB04ED.exa MB04FD.exa MB04GD.exa \
      MB04MD.exa MB04OD.exa MB04PB.exa MB04PU.exa MB04TB.exa MB04TS.exa \
-     MB04UD.exa MB04VD.exa MB04XD.exa MB04YD.exa MB04ZD.exa MB05MD.exa \
+     MB04UD.exa MB04XD.exa MB04YD.exa MB04ZD.exa MB05MD.exa \
      MB05ND.exa MB05OD.exa MC01MD.exa MC01ND.exa MC01OD.exa MC01PD.exa \
      MC01QD.exa MC01RD.exa MC01SD.exa MC01TD.exa MC01VD.exa MC01WD.exa \
-     MC01XD.exa MC03MD.exa MC03ND.exa MD03AD.exa MD03BD.exa \
-     SB01BD.exa SB01DD.exa SB01MD.exa SB02MD.exa SB02ND.exa SB02OD.exa \
+     MC01XD.exa MC03MD.exa MD03AD.exa \
+     SB01BD.exa SB01MD.exa SB02MD.exa SB02ND.exa SB02OD.exa \
      SB02PD.exa SB02QD.exa SB02RD.exa SB02SD.exa SB03MD.exa SB03OD.exa \
      SB03QD.exa SB03SD.exa SB03TD.exa SB03UD.exa SB04MD.exa SB04ND.exa \
-     SB04OD.exa SB04PD.exa SB04QD.exa SB04RD.exa SB06ND.exa SB08CD.exa \
+     SB04OD.exa SB04PD.exa SB04QD.exa SB04RD.exa SB08CD.exa \
      SB08DD.exa SB08ED.exa SB08FD.exa SB08MD.exa SB08ND.exa SB09MD.exa \
-     SB10DD.exa SB10ED.exa SB10FD.exa SB10HD.exa SB10ID.exa SB10KD.exa \
-     SB10ZD.exa SB16AD.exa SB16BD.exa SB16CD.exa SG02AD.exa SG02ND.exa \
+     SB10DD.exa SB10ED.exa SB10FD.exa SB10HD.exa SB10KD.exa \
+     SB10ZD.exa SB16AD.exa SB16BD.exa SB16CD.exa SG02ND.exa \
      SG03AD.exa SG03BD.exa \
      TB01ID.exa TB01IZ.exa TB01KD.exa TB01LD.exa TB01MD.exa TB01ND.exa \
-     TB01PD.exa TB01PX.exa TB01TD.exa TB01UD.exa TB01UY.exa TB01WD.exa \
-     TB01WX.exa TB01ZD.exa TB03AD.exa TB04AD.exa TB04BD.exa TB04CD.exa \
-     TB05AD.exa TC01OD.exa TC04AD.exa TC05AD.exa TD03AD.exa TD04AD.exa \
+     TB01TD.exa TB01UY.exa TB01WD.exa \
+     TB01WX.exa TB01ZD.exa TB04BD.exa TB04CD.exa \
+     TC01OD.exa TC04AD.exa TC05AD.exa \
      TD05AD.exa TF01MD.exa TF01ND.exa TF01OD.exa TF01PD.exa TF01QD.exa \
      TF01RD.exa TG01AD.exa TG01AZ.exa TG01CD.exa TG01DD.exa TG01ED.exa \
      TG01FD.exa TG01FZ.exa TG01GD.exa TG01HD.exa TG01ID.exa TG01JD.exa \
